docs/io/fits/usage/headers.rst

```diff
@@ -336,8 +336,7 @@ for keywords, a proposal was made in:
     https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/ofwg_recomm/r13.html
 
 by using the CONTINUE keyword after the regular 80 column containing the
-keyword. ``astropy`` does support this convention, even though it is not a FITS
-standard.
+keyword. ``astropy`` does support this convention, which is a part of the FITS standard since version 4.0.
```

Comment:
This looks much better now. However we generally require lines to be <= 80 characters long. Could you include a newline between "FITS" and "standard"?

Hint: For documentation-only changes you can add `[docs only]` to the commit message so the test-infrastructure knows that it doesn't have to run the full-test-suite.

If possible amend it to your latest commit - but if you don't have much experience with git and don't know how to amend then feel free to ignore this.